backend/Core/DomainModels/Consultant.cs

```diff
@@ -38,6 +39,36 @@ public int YearsOfExperience
             return currentAcademicYear - GraduationYear ?? currentAcademicYear;
         }
     }
+
+    public double HoursPrWorkDay => Department.Organization.HoursPerWorkday;
+
+    public double GetVacationHoursForWeek(Week week)
```

Comment:
Does this one handle Year? What about If theres is set vacation in week 50 next year too?

Reply:
From DateIsInWeek: 
```
    public static bool DateIsInWeek(DateOnly day, Week week)
    {
        return day.Year == week.Year && GetWeekNumber(day.ToDateTime(TimeOnly.MinValue)) == week.WeekNumber;
    }
```
Yes, it checks for year